.devcontainer/ubuntu18.04/devcontainer.json

```diff
@@ -1,9 +1,38 @@
+// For format details, see https://aka.ms/devcontainer.json.
 {
   "name": "Homebrew/brew-ubuntu18.04",
   "image": "ghcr.io/homebrew/ubuntu18.04:latest",
+
   "workspaceFolder": "/home/linuxbrew/.linuxbrew/Homebrew",
   "workspaceMount": "source=${localWorkspaceFolder},target=/home/linuxbrew/.linuxbrew/Homebrew,type=bind,consistency=cached",
+
   "onCreateCommand": ".devcontainer/on-create-command.sh",
+
+  "customizations": {
+    "vscode": {
+      "settings": {
+        // Turns off the display of CPU and battery frequency (because it's useless)
+        // This setting is only required for GitHub Codespaces
+        "resmon.show.battery": false,
```

Comment:
I don't think it hurts, really. Let's not disable those and keep the configuration as simple as possible.

Reply:
Done - [`39aaf33` (#17298)](https://github.com/Homebrew/brew/pull/17298/commits/39aaf333403e8b4d636ff6fc08650e9f67f9ff77)